### PR TITLE
202109 Use different comments depending on is_main

### PR DESCRIPTION
**Pull request by owenThurm**
Use comments that send to your "Send to brand" if `is_main = False`.

Will add validation and a reusable function for validating/fetching GUI input data in the next PR

### CODE REVIEW

File: src/GUI/bot_gui.py [hunk 1]
```diff
@@ -23,6 +23,20 @@
     ON_SWITCH_FILE_PATH,
     OFF_SWITCH_FILE_PATH,
     GA_LOGO_FILE_PATH,
+    GUI_THEME,
+    CONFIG_FILE,
+    CONFIG_USERNAME,
+    CONFIG_PASSWORD,
```
Comment (DanielGelfand) [inline src/GUI/bot_gui.py:29]: Awesome.

File: src/utility_methods.py [hunk 2]
```diff
@@ -2,108 +2,108 @@
...
+        f"Dope shot! dm @{brand_name} when you can",
+        f"Dope shot! dm @{brand_name} whenever you get the chance!",
+        f"Dope shot! dm @{brand_name} when you get the chance",
+        f"Dope shot! when you get the chance shoot a dm to @{brand_name}",
+        f"Dope shot! when you’re free shoot a dm to @{brand_name}",
+        f"Dope shot! when you’re free dm @{brand_name}!",
+        f"Dope shot! when you get some time shoot a dm to @{brand_name}",
+        f"Dope shot! shoot a dm to @{brand_name} when you get the time",
+        f"Dope shot! shoot a dm to @{brand_name} when you get a chance",
+        f"Dope shot! shoot a dm to @{brand_name} when you get the chance"
```
Comment (DanielGelfand) [inline src/utility_methods.py:44]: Needs comma here

File: src/utility_methods.py [hunk 3]
```diff
@@ -2,108 +2,108 @@
...
+        f"Dope shot fr! whenever you're free shoot a dm to @{brand_name}",
+        f"Dope shot! dm @{brand_name} whenever you’re free!",
+        f"Vibes! shoot @{brand_name} a dm whenever you get the chance",
+        f"Vibes! when you get a chance shoot a dm to @{brand_name}",
+        f"Vibes! shoot a dm to @{brand_name} when you can",
+        f"Vibes! dm @{brand_name} whenever you get the time",
+        f"Vibes fr! dm @{brand_name} whenever you get the chance",
+        f"Vibes fr! dm @{brand_name} whenever you're free!",
+        f"Vibes fr! dm @{brand_name} when you find the time!",
+        f"Vibes! if you get a chance shoot a dm to @{brand_name}"
```
Comment (DanielGelfand) [inline src/utility_methods.py:56]: Comma here

File: src/utility_methods.py [hunk 4]
```diff
@@ -226,29 +226,3 @@ def random_comment_text_main() -> str:
...
-    comments = [
-        "It’s a vibe🙌 shoot @genuineaesthetic a dm when you can🙏",
-        "It’s a vibe🙌 dm @genuineaesthetic when you can🙏 ",
-        "It’s a vibe👌 when you get the chance shoot @genuineaesthetic a dm🙏",
-        "It’s a vibe🤟when you get the chance shoot a dm to @genuineaesthetic 🙏",
-        "It’s a vibe🤟shoot a dm to @genuineaesthetic when you can🙌",
-        "It’s a vibe👌when you’re free shoot a dm to @genuineaesthetic 🤟",
-        "It’s a vibe🙌 dm @genuineaesthetic whenever you’re free🙏",
-        "It’s a vibe🙏 whenever you get a minute dm @genuineaesthetic 🙌",
-        "It’s a vibe🙌 whenever you get a minute shoot @genuineaesthetic a dm🙏",
```
Comment (DanielGelfand) [inline src/utility_methods.py:241]: About time we removed  this.

File: src/utility_methods.py (hunk 2 above)
Comment (owenThurm) [reply src/utility_methods.py:44]: nice catch

File: src/utility_methods.py (hunk 3 above)
Comment (owenThurm) [reply src/utility_methods.py:56]: nice catch

File: src/utility_methods.py (hunk 2 above)
Comment (owenThurm) [reply src/utility_methods.py]: Should have got them all now, used cmd f to search the file to make sure

**Review by DanielGelfand — APPROVED** [review]

**Pull request merged**